server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/QProfileTester.java

```diff
@@ -49,7 +51,11 @@
     this.session = session;
   }
 
-  public QualityProfilesService service() {
+  public QualityProfilesService serviceOld() {
+    return session.wsClient().qualityProfilesOld();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "qualityProfilesOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)